package-lock.json

```diff
@@ -4920,8 +4920,25 @@
             "from": "git://github.com/mathjax/MathJax-src.git",
             "requires": {
                 "esm": "^3.2.25",
-                "mj-context-menu": "^0.2.0",
-                "speech-rule-engine": "^3.0.0-beta.6"
+                "mj-context-menu": "^0.2.2",
+                "speech-rule-engine": "^3.0.0-beta.8"
+            },
+            "dependencies": {
+                "mj-context-menu": {
+                    "version": "0.2.2",
+                    "resolved": "https://registry.npmjs.org/mj-context-menu/-/mj-context-menu-0.2.2.tgz",
+                    "integrity": "sha512-OHlnKQqfFPEYZGdz2JWL0obrr82vVilha0WCUZslYfN+v+oz4VpmERnoHdTUWvOUVHNYjFkpOYnLEeHnt1BdsQ=="
+                },
+                "speech-rule-engine": {
+                    "version": "3.0.0-beta.8",
+                    "resolved": "https://registry.npmjs.org/speech-rule-engine/-/speech-rule-engine-3.0.0-beta.8.tgz",
+                    "integrity": "sha512-gnNap93Xrz5ngIqa/BOaJGatC64B7dqDrXK/pr4yAaZAcW6tKVO5PwaTvw9QqVab86ad5scpA39Nxwhlb11tTg==",
+                    "requires": {
+                        "commander": "*",
+                        "wicked-good-xpath": "*",
+                        "xmldom-sre": "^0.1.31"
+                    }
+                }
```

Comment:
Why do we have changes to `package-lock.json`? Are these needed?

Reply:
sorry, I think we changed this file by accident, can you just remove the changes we did for this file? thanks !!!

Reply:
Can you revert these, as PR author? You can use git commands to revert them.